util/android-commands.sh

```diff
@@ -209,7 +210,7 @@ snapshot() {
     # We need to install nextest via cargo currently, since there is no pre-built binary for android x86
     command="'\
 export CARGO_TERM_COLOR=always; \
-cargo install cargo-nextest; \
+#cargo install cargo-nextest; \
```

Comment:
Is the rest of this block still necessary? It might also be useful to add a hint why it is commented out.